sys-fs/cryfs/cryfs-0.9.7.ebuild

```diff
@@ -0,0 +1,75 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=5
+PYTHON_COMPAT=( python{2_7,3_{4,5,6}} )
+
+inherit cmake-utils python-any-r1
+
+DESCRIPTION="Encrypted FUSE filesystem that conceals metadata"
+HOMEPAGE="https://www.cryfs.org/"
+
+SLOT=0
+IUSE="-update-check -test -debug"
+
+LICENSE="LGPL-3 BSD-2 MIT
+	test? ( BSD )"
+# cryfs - LGPL-3
+# scrypt - BSD-2
+# spdlog - MIT
```

Comment:
Yes, it unfortunately bundles scrypt and spdlog. I've requested upstream that CMake options be added to use the system versions instead. If that isn't done then I'll look at doing it myself in the ebuild. It should be easy to replace spdlog with the system version, but to use the system scrypt, the app-crypt/scrypt ebuild would need to be modified to install some header/source files, as right now it only installs the binary /usr/bin/scrypt.